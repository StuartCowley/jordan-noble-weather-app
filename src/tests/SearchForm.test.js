import React from "react";
import { render, screen } from "@testing-library/react";
import SearchForm from "../components/SearchForm";

describe("SearchForm", () => {
  it("renders correctly", () => {
    const { asFragment, getByText } = render(<SearchForm />);

    expect(asFragment()).toMatchSnapshot();
    expect(getByText("Search")).toBeInTheDocument();
  });
});
