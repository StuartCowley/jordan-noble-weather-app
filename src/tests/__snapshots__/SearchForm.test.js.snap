// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders correctly 1`] = `
<DocumentFragment>
  <div
    class="search-form"
  >
    <input
      class="search-form_input"
      type="text"
    />
    <button
      class="search-form_button"
      type="submit"
    >
      Search
    </button>
  </div>
</DocumentFragment>
`;
